### DISCUSSION TITLE
build failed of diesel v1.4.7

### DISCUSSION THREAD

**Discussion by q0wert** [diesel-rs/diesel#2839] [Q&A]
i followed steps of the graph (https://github.com/graphprotocol/graph-node)

install rust and clone from https://github.com/graphprotocol/graph-node, and run cargo build.

when compiling diesel v1.4.7, it failed.

`process didn't exit successfully: `rustc --crate-name diesel /root/.cargo/registry/src/github.com-1ecc6299db9ec823/diesel-1.4.7/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts --crate-type lib --emit=dep-info,metadata,link -C embed-bitcode=no -C debuginfo=2 --cfg 'feature="32-column-tables"' --cfg 'feature="bigdecimal"' --cfg 'feature="bitflags"' --cfg 'feature="chrono"' --cfg 'feature="default"' --cfg 'feature="num-bigint"' --cfg 'feature="num-integer"' --cfg 'feature="num-traits"' --cfg 'feature="numeric"' --cfg 'feature="postgres"' --cfg 'feature="pq-sys"' --cfg 'feature="r2d2"' --cfg 'feature="serde_json"' --cfg 'feature="with-deprecated"' -C metadata=d11e034628f8c4cd -C extra-filename=-d11e034628f8c4cd --out-dir /root/graph-node/target/debug/deps -L dependency=/root/graph-node/target/debug/deps --extern bigdecimal=/root/graph-node/target/debug/deps/libbigdecimal-a0cc5b12bec2c79f.rmeta --extern bitflags=/root/graph-node/target/debug/deps/libbitflags-f89ae4b715176e2d.rmeta --extern byteorder=/root/graph-node/target/debug/deps/libbyteorder-d836ba3e41654e24.rmeta --extern chrono=/root/graph-node/target/debug/deps/libchrono-bfd37b14c109812d.rmeta --extern diesel_derives=/root/graph-node/target/debug/deps/libdiesel_derives-d44d0ffbdc66e423.so --extern num_bigint=/root/graph-node/target/debug/deps/libnum_bigint-f45e744821fd24e9.rmeta --extern num_integer=/root/graph-node/target/debug/deps/libnum_integer-14a43334321bcbad.rmeta --extern num_traits=/root/graph-node/target/debug/deps/libnum_traits-d2f26dc32356e531.rmeta --extern pq_sys=/root/graph-node/target/debug/deps/libpq_sys-d12374d028952095.rmeta --extern r2d2=/root/graph-node/target/debug/deps/libr2d2-b7e7fbbfb9fb9cd7.rmeta --extern serde_json=/root/graph-node/target/debug/deps/libserde_json-98e3d42a1be7a869.rmeta --cap-lints allow` (signal: 9, SIGKILL: kill)`

any help is appreciated.

**Comment by weiznich** [comment]
The error message indicates that your rust compiler is stoped by an external signal. This is not related to diesel. Please consider to ask the authors of `graph-node` for help.
